packages/shared/src/styles/base.css

```diff
@@ -798,3 +798,7 @@ details.right-icon {
   grid-column-gap: 0.5rem;
   padding-bottom: max(env(safe-area-inset-bottom, 0.5rem), 0.5rem);
 }
+
+.link {
+  @apply text-text-link underline hover:no-underline;
+}
```

Comment:
Why don't we make this a component? 
It will be easier to also add other things down the line.

Reply:
Sure, we can maybe make it similar to the `Image` component.

Reply:
Update 🚀